osdocs-3441:Adding OpenShift Local Installation Overview (#61058)
openshift/openshift-docs: modules/installation-openshift-local.adoc

// Module included in the following assemblies:
//
// * getting_started/openshift-overview.adoc
// * installing/index.adoc

:_content-type: CONCEPT
[id="installation-openshift-local_{context}"]

> = {product-title} Local Overview

Reviewer: ```suggestion
= {product-title} Local overview
```

> Red Hat {product-title} Local supports rapid application development to get started building {product-title} clusters. {product-title} Local is designed to run on a local computer to simplify setup and testing, and to emulate the cloud development environment locally with all of the tools needed to develop container-based applications. 

Reviewer: ```suggestion
{product-title} Local supports rapid application development to get started building {product-title} clusters. {product-title} Local is designed to run on a local computer to simplify setup and testing, and to emulate the cloud development environment locally with all of the tools needed to develop container-based applications. 
```

> Whatever your programming language, {product-title} Local hosts your application and brings a minimal, preconfigured Red Hat {product-title} 4.1 or newer cluster to your local PC without the need for a server-based infrastructure.

Reviewer: ```suggestion
Regardless of the programming language you use,  {product-title} Local hosts your application and brings a minimal, preconfigured {product-title} 4.1 or newer cluster to your local PC without the need for a server-based infrastructure.
```

Not a huge fan of "Whatever." There's only a few instances of it in the docs (26). Feel free to play with the wording, or ignore my suggestion altogether.

Author: Thanks for bringing this up! I decided to do away with the product version (4.1) for the sake of clarity. I think that the sentence still conveys that OpenShift Local brings a preconfigured OpenShift cluster to a local device. Feel free to correct me if I'm wrong. 

Just another note that I changed the wording from {product-title} Local to "OpenShift Local" because the former translated to "OpenShift Container Platform Local" and that terminology is not used in its marketing communication. Just to keep terms consistent, I decided to only include {product-title} when referring to clusters. Let me know too if this seems incorrect, or if you have any questions. 

Thanks for your time on this. :)

> On a hosted environment, {product-title} Local supports creating microservices, building microservices into images, and running them in Kubernetes-hosted containers right on your laptop or desktop running Linux, macOS, or Windows 10. 

Reviewer 2: Windows 10 and later.

> On a hosted environment, {product-title} Local supports creating microservices, building microservices into images, and running them in Kubernetes-hosted containers right on your laptop or desktop running Linux, macOS, or Windows 10 and later. 

Reviewer: Slight suggestion to reduce the redundancy of "running"

```suggestion
On a hosted environment, with {product-title} Local, you can create microservices, convert them into images, and run them in Kubernetes-hosted containers directly on your laptop or desktop running Linux, macOS, or Windows 10 or later.
```

Feel to ignore/revise another way!

// Module included in the following assemblies:
//
// * getting_started/openshift-overview.adoc
// * installing/index.adoc

:_content-type: CONCEPT
[id="installation-openshift-local_{context}"]
= {product-title} Local Overview

> Red Hat {product-title} Local (formerly Red Hat CodeReady Containers) supports rapid application development to get started building {product-title} clusters. {product-title} Local is designed to run on a local computer to simplify setup and testing, and to emulate the cloud development environment locally with all of the tools needed to develop container-based applications. 

Reviewer 2: Not sure if you still need to reference `(formerly Red Hat CodeReady Containers)`

Whatever your programming language, {product-title} Local hosts your application and brings a minimal, preconfigured Red Hat {product-title} 4.1 or newer cluster to your local PC without the need for a server-based infrastructure.

> With {product-title} Local, you can create microservices, build them into images, and run them in Kubernetes-hosted containers, right on your laptop or desktop running Linux, macOS, or Windows 10. 

Reviewer 2: `With` is weird, as `you can create microservices` and `build them into images` is not what we do. It is easily mis-read. We merely 'host' these applications

Author: Thanks for your review! I have updated this sentence to include some "hosting" language, please let me know if it makes sense or needs revision. Appreciate your time!

> With {product-title} Local, you can create microservices, build them into images, and run them in Kubernetes-hosted containers, right on your laptop or desktop running Linux, macOS, or Windows 10. For more information, see link:https://developers.redhat.com/products/openshift-local/overview[Red Hat {product-title} Local Overview].

Reviewer 2: Duplcated?
